bbc/simorgh
# PREVIEW: Use `simorgh1` for static assets path

Resolves JIRA [number]

Overall changes
======
Update the static assets path for the PREVIEW environment to
adopt the new `simorgh1` domain.

Code changes
======

- Update `SIMORGH_PUBLIC_STATIC_ASSETS_PATH` to use `/ws/simorgh1-preview-assets/public/`

Testing
======
1. Visit https://static.test.files.bbci.co.uk/ws/simorgh-preview-assets/public/static/js/modern.main-d91a9049.becde327.js
2. The URL above should `404`.
3. Visit https://static.test.files.bbci.co.uk/ws/simorgh1-preview-assets/public/static/js/modern.main-d91a9049.becde327.js
4. The URL should `200`.

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
